universal-application-tool-0.0.1/app/repository/QuestionRepository.java

```diff
@@ -135,6 +143,22 @@ private void updateAllRepeatedQuestions(long newEnumeratorId, long oldEnumerator
     return conflictDetector.getConflictedQuestion();
   }
 
+  public ImmutableList<QuestionDefinition> getAllQuestionsForTag(QuestionTag tag) {
```

Comment:
suggestion (blocking): please document this public method and include whether it's *all* questions or just the ones in the active version

Reply:
Done.